src/convertToObject.js

```diff
@@ -5,8 +5,21 @@
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
   // write your code here
+  const stylesArray = sourceString
+    .split(';')
+    .map((item) => item.split(':').map((part) => part.trim()))
+    .filter((item) => item.length === 2 && item[0] && item[1]);
```

Comment:
The chaining of methods here is correct, but the indentation is not consistent with the best practices for readability. Each method in the chain should start on a new line with the same indentation level as the first one.